dev/src/DocFx/Node/ClassNode.php

```diff
@@ -164,6 +164,11 @@ public function getMethods(): array
             }
         }
 
+        if ($this->isServiceClass()) {
```

Comment:
Just wanna confirm, this is only run in PHP version greater than `8.0`, right?
Because, prior to that, the original order of usort isn't preserved, I think.

Reply:
Yes that's correct, everything in the `dev` directory requires PHP 8.1

see [dev/composer.json](https://github.com/googleapis/google-cloud-php/blob/main/dev/composer.json#L5)